### PR TITLE
Change query expression to foreach

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
1. $subject
2. Changed concurrency to 1 in bidirectional scenario(since the bidi rpc contains a lock statement)

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2663

### CODE REVIEW

File: load-tests/route_guide_bidirectional_streaming/scripts/run.sh [hunk 1]
```diff
@@ -20,5 +20,5 @@ set -e
 source base-scenario.sh
 
 echo "----------Running load tests----------"
-./ghz --skipTLS --proto $scriptsDir/route_guide.proto --rps 500 --duration 1800s --concurrency 10 --connections 10 --skipFirst 1000 --duration-stop wait --call routeguide.RouteGuide.RouteChat --stream-call-count=5 -d '[{"location": {"latitude": 406109563, "longitude": -742186778}, "message": "m1"},{"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}, {"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}]' bal.perf.test:443 --format influx-summary --output $scriptsDir/ghz_output.csv
+./ghz --skipTLS --proto $scriptsDir/route_guide.proto --rps 500 --duration 1800s --concurrency 1 --connections 1 --skipFirst 1000 --duration-stop wait --call routeguide.RouteGuide.RouteChat --stream-call-count=5 -d '[{"location": {"latitude": 406109563, "longitude": -742186778}, "message": "m1"},{"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}, {"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}]' bal.perf.test:443 --format influx-summary --output $scriptsDir/ghz_output.csv
```
Comment (BuddhiWathsala) [inline load-tests/route_guide_bidirectional_streaming/scripts/run.sh:23]: Do we still need to reduce the connection count even after the query expression change?

Comment (dilanSachi) [reply load-tests/route_guide_bidirectional_streaming/scripts/run.sh:23]: No. This is for the bidirectional case. It gives around 0.5% error rate with 10 connections.

Comment (BuddhiWathsala) [reply load-tests/route_guide_bidirectional_streaming/scripts/run.sh:23]: Isn't there a number between 1 < n < 10, that we can stick to?.

Comment (dilanSachi) [reply load-tests/route_guide_bidirectional_streaming/scripts/run.sh:23]: Yes need to find the optimum connection count. But what I had in mind was to first make sure that this n=1 completes without any error rate. Then I can test in the local cluster for the optimum number

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**